### PR TITLE
add app-drawn windows titlebar

### PR DESCRIPTION
**Pull request by notAlaanor**
| Q                | A
| ---------------- | ---
| New feature?     | yes

### Description
App is frameless for Windows, and window controls are drawn by the application instead of being native Windows. Menu is now a popup in the corner for Windows.

### CODE REVIEW

**Comment by Jocs** [comment]
@notAlaanor 3q, I'll review the PR after I fix the [Travis CI build failed](https://travis-ci.org/marktext/marktext/builds/351909280?utm_source=github_status&utm_medium=notification), If you have time, could you pls help me find the reason why this failed.

**Comment by Jocs** [comment]
@notAlaanor has conflicts, pls pull the latest commit, and update this PR. Thank you.

**Comment by Jocs** [comment]
@notAlaanor hi, notAlaanor, I add you avatar in the readme file, If you don't want to, I'll delete it.

**Comment by notAlaanor** [comment]
@Jocs, it's fine, I don't mind. 👍

File: src/renderer/store/editor.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ const state = {
     word: 0,
     character: 0,
     all: 0
-  }
+  },
+  platform: ''
```
Comment (Jocs) [inline src/renderer/store/editor.js]: platform:  process.platform

Set the init value, So, you don't need to dispatch the action 'GET_PLATFORM'.

**Comment by Jocs** [comment]
@notAlaanor great！

**Pull request merged**